man/systemd.network.xml

```diff
@@ -374,11 +374,12 @@
         <varlistentry>
           <term><varname>IPv6Token=</varname></term>
           <listitem>
-            <para>An IPv6 address with the top 64 bits unset. When set, indicates the
-            64-bit interface part of SLAAC IPv6 addresses for this link. Note that
-            the token is only ever used for SLAAC, and not for DHCPv6 addresses, even
-            in the case DHCP is requested by router advertisement. By default, the
-            token is autogenerated.</para>
+            <para>Taken one of IPv6 address with the top 64 bits unset or prefixstable.
+            When IPv6 address is set, indicates the 64-bit interface part of SLAAC IPv6
+            addresses for this link. Note that the token is only ever used for SLAAC,
+            and not for DHCPv6 addresses, even in the case DHCP is requested by router
+            advertisement. By default, the token is autogenerated. When set to prefixstable
+            then see [IPv6Token] section.</para>
```

Comment:
`When set to <literal>prefixstable</literal>, some short explanation here, see [IPv6Token] section.`